### ISSUE TITLE
New translation: Hebrew (Ivrit)

### ISSUE THREAD

**Issue by dennisvanlith** [squidfunk/mkdocs-material#721]
## Instructions

1. Check, if your language is already available: [here](http://bit.ly/2DCzaL0)
2. If it isn't, please translate the labels on the right:

``` jinja
{% macro t(key) %}{{ {
  "language": "he",
  "direction": "rtl",
  "clipboard.copy": "העתק ללוח",
  "clipboard.copied": "הועתק ללוח",
  "edit.link.title": "ערוך דף זה",
  "footer.previous": "קודם",
  "footer.next": "הַבָּא",
  "meta.comments": "הערות",
  "meta.source": "מָקוֹר",
  "search.language": "he",
  "search.pipeline.stopwords": true,
  "search.pipeline.trimmer": true,
  "search.placeholder": "לחפש",
  "search.result.placeholder": "הקלד כדי להתחיל לחפש",
  "search.result.none": "אין מסמכים תואמים",
  "search.result.one": "1 מסמך תואם",
  "search.result.other": "# מסמך תואם",
  "search.tokenizer": "[\s\-]+",
  "skip.link.title": "דלג לתוכן",
  "source.link.title": "עבור אל מאגר",
  "toc.title": "תוכן העניינים"
}[key] }}{% endmacro %}
```
Added hebrew language for MKDocs

Thanks!

**Comment by squidfunk** [comment]
Awesome, thanks!

**Comment by squidfunk** [comment]
Fixed and released as part of 2.7.0
